features/step_definitions/custom_web_steps.rb

```diff
@@ -142,6 +142,15 @@ def take_screenshots_with_login
   page.driver.browser.switch_to.alert.dismiss
 end
 
+And /^I should not see any alert$/ do
+  begin
+    page.driver.browser.switch_to.alert
+  rescue Selenium::WebDriver::Error::NoAlertPresentError
+    true
+  end
+  false
+end
```

Comment:
Well, returning true or false is meaningless. Ideally you would turn this into an RSpec matcher: https://relishapp.com/rspec/rspec-expectations/v/3-3/docs/custom-matchers/define-matcher